fab/pillows/swiss.yml

```diff
@@ -5,3 +5,5 @@ exclude_pillows:
   - ReportCaseToElasticsearchPillow
   - ReportXFormToElasticsearchPillow
   - CaseSearchToElasticsearchPillow
+  - kafka-ucr-static
+  - kafka-ucr-main
```

Comment:
yeah i've always wondered about that as well. i don't think it'd be too too hard, but this might do the trick as here are the current processes that take up a lot of mem:

<img width="1416" alt="screen shot 2016-09-13 at 5 29 36 pm" src="https://cloud.githubusercontent.com/assets/918514/18492127/b10fa4de-79d7-11e6-9c27-c925568f94d8.png">

kafka-ucr-static will be eliminated (#3)
formplayer.jar will be eliminated (#4)

after that list is a bunch of celery which will hopefully be eased with autoscale. lastly i think we could cut down on ES mem if we're still having troubles. i think if this doesn't help, we should definitely implement the pillow-key thing